src/AwsLambda/AwsLambdaWrapper/Helpers/ServiceHelpers.cs

```diff
@@ -37,11 +37,10 @@ internal static ISpan CreateSpan(KeyValuePair<string, string> operationPair, str
         /// <returns></returns>
         internal static ISpan CreateSpan(string operationName, string component, string operation)
         {
-            if (string.IsNullOrEmpty(operationName) || string.IsNullOrEmpty(component) || string.IsNullOrEmpty(operation))
-            {
-                Logger.Log("Could not create a span. Creating a span requires an operation name, component, and operation.", true, "ERROR");
-                return null;
-            }
+
+            operationName = string.IsNullOrEmpty(operationName) ? "UNKNOWN" : operationName;
```

Comment:
Any time a string gets used multiple times, you might want to consider making it a constant, for readability.

Reply:
Resolves this as you already approved this PR.